voice/receive-an-inbound-call.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/vonage/vonage-go-sdk/ncco"
+)
+
+func answer(w http.ResponseWriter, req *http.Request) {
+
+	MyNcco := ncco.Ncco{}
+
+	talk := ncco.TalkAction{Text: "Thank you for calling."}
+	MyNcco.AddAction(talk)
+
+	data, _ := json.Marshal(MyNcco)
+	fmt.Fprintf(w, "%s", data)
```

Comment:
Since this a web call, we should write the JSON to the response instead of splatting out the JSON directly (https://www.alexedwards.net/blog/golang-response-snippets#json)